doc/source/configuration/monitoring.rst

```diff
@@ -10,4 +14,52 @@ The configuration options can be found in
 ``etc/kayobe/stackhpc-monitoring.yml``:
 
 .. literalinclude:: ../../../etc/kayobe/stackhpc-monitoring.yml
-   :language: yaml
\ No newline at end of file
+   :language: yaml
+
+In order to enable stock monitoring configuration within a particular
+environment, create the following symbolic links:
+
+.. code-block:: console
+
+    cd $KAYOBE_CONFIG_PATH
+    ln -s kolla/config/grafana/ environments/$KAYOBE_ENVIRONMENT/kolla/config/
+    ln -s kolla/config/prometheus/ environments/$KAYOBE_ENVIRONMENT/kolla/config/
+
+and commit them to the config repository.
```

Comment:
There's also the rsyslog custom playbook. How and when should it be used?

Reply:
Added